scss/pages/blog-post.scss

```diff
@@ -0,0 +1,124 @@
+@import "common/base/topic-post";
+
+body.blog-post {
+  background-image: unset;
```

Comment:
@MeghnaAJ Let's see if we can [@extend](https://sass-lang.com/documentation/at-rules/extend) the core discourse classes here.

Reply:
I'm having hard time using extend for `<blockquote>` and `<code>` tags because it seems like `@extend` only works for classes.